packages/blade/src/storybook-recipes/ThemePlayground/BrandedComponentKitchenSink.tsx

```diff
@@ -0,0 +1,94 @@
+import { composeStories } from '@storybook/react';
+import * as accordionStories from '../../components/Accordion/Accordion.stories';
+import * as buttonStories from '../../components/Button/Button/Button.stories';
+import * as bottomSheetStories from '../../components/BottomSheet/BottomSheet.stories';
+import * as carouselStories from '../../components/Carousel/Carousel.stories';
+import * as checkboxGroupStories from '../../components/Checkbox/CheckboxGroup.stories';
+import * as chipGroupStories from '../../components/Chip/ChipGroup.stories';
+import * as collapsibleStories from '../../components/Collapsible/Collapsible.stories';
+import * as modalStories from '../../components/Modal/docs/SimpleModal.stories';
+import * as progressBarStories from '../../components/ProgressBar/ProgressBar.stories';
+import * as radioStories from '../../components/Radio/Radio.stories';
+import * as switchStories from '../../components/Switch/Switch.stories';
+import * as textInputStories from '../../components/Input/TextInput/TextInput.stories';
+import * as otpInputStories from '../../components/Input/OTPInput/OTPInput.stories';
+import * as autoCompleteStories from '../../components/Input/DropdownInputTriggers/AutoComplete.stories';
+import { Card, CardBody } from '~components/Card';
+import { Box } from '~components/Box';
+import { Heading } from '~components/Typography';
+import { Divider } from '~components/Divider';
+
+const includedStories = [
+  'IconRightButton',
+  'WithIcons',
+  'WithHeaderFooter',
+  'DefaultCarousel',
+  'HelpTextCheckbox',
+  'DefaultSelectedSingle',
+  'WithCollapsibleButton',
+  'SimpleModal',
+  'ProgressBarIndeterminate',
+  'RequiredRadio',
+  'DefaultCheckedSwitch',
+  'TextInputHelpText',
+  'OTPInputHelpText',
+  'InternalAutoCompleteControlled',
+];
+
+const StoryNameToHeadingMap = {
+  IconRightButton: 'Button',
+  WithIcons: 'Accordion',
+  WithHeaderFooter: 'BottomSheet',
+  DefaultCarousel: 'Carousel',
+  HelpTextCheckbox: 'Checkbox',
+  DefaultSelectedSingle: 'Chip',
+  WithCollapsibleButton: 'Collapsible',
+  SimpleModal: 'Modal',
+  ProgressBarIndeterminate: 'ProgressBar',
+  RequiredRadio: 'Radio',
+  DefaultCheckedSwitch: 'Switch',
+  TextInputHelpText: 'TextInput',
+  OTPInputHelpText: 'OTPInput',
+  InternalAutoCompleteControlled: 'AutoComplete',
+};
+
+const allStories = [];
```

Comment:
nit: why do you need to call push? you can directly add values to `allStories` array?

```
const allStories = [
...Object.values(composeStories(buttonStories))
]
```
like this?